README.md

```diff
@@ -61,6 +61,46 @@ PyPI, which is recommended to restrict the access the action has.
 The secret used in `${{ secrets.PYPI_API_TOKEN }}` needs to be created on the
 settings page of your project on GitHub. See [Creating & using secrets].
 
+### Publishing with OpenID Connect
+
+**IMPORTANT**: This functionality is in beta, and will not work for you
+unless you're a member of the PyPI OIDC beta testers' group. For more
+information, see
+[warehouse#12965](https://github.com/pypi/warehouse/issues/12965).
```

Comment:
Plz convert this into a detached reference at the end of the document, following the rest of the links.